modules/grpc/metadata.json

```diff
@@ -2,70 +2,8 @@
     "homepage": "https://github.com/grpc/grpc",
     "maintainers": [
         {
-            "email": "veblush@google.com",
-            "github": "veblush",
-            "name": "Esun Kim",
-            "github_user_id": 2793282
-        },
-        {
-            "email": "bpawan@google.com",
-            "github": "pawbhard",
-            "name": "Pawan Bhardwaj",
-            "github_user_id": 13415978
-        },
-        {
-            "email": "eostroukhov@google.com",
-            "github": "eugeneo",
-            "name": "Eugene Ostroukhov",
-            "github_user_id": 287917
-        },
-        {
-            "email": "hork@google.com",
-            "github": "drfloob",
-            "name": "Adam Heller",
-            "github_user_id": 295906
-        },
-        {
-            "email": "rishesh@google.com",
-            "github": "rishesh007",
-            "name": "Rishesh Agarwal",
-            "github_user_id": 40896662
-        },
-        {
-            "email": "roth@google.com",
-            "github": "markdroth",
-            "name": "Mark Roth",
-            "github_user_id": 18664614
-        },
-        {
-            "email": "yashkt@google.com",
-            "github": "yashykt",
-            "name": "Yash Tibrewal",
-            "github_user_id": 4181124
-        },
-        {
-            "email": "apolcyn@google.com",
-            "github": "apolcyn",
-            "name": "Alexander Polcyn",
-            "github_user_id": 9566254
-        },
-        {
-            "email": "sergiitk@google.com",
-            "github": "sergiitk",
-            "name": "Sergii Tkachenko",
-            "github_user_id": 672669
-        },
-        {
-            "email": "ssreenithi@google.com",
-            "github": "sreenithi",
-            "name": "Sreenithi Sridharan",
-            "github_user_id": 22791051
-        },
-        {
-            "email": "stanleycheung@google.com",
-            "github": "stanley-cheung",
-            "name": "Stanley Cheung",
-            "github_user_id": 11674202
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
This reverts #5244. Is this intentional?

Reply:
Good catch. It shouldn't be like this. gRPC repo also needs the same change to prevent this kind of revert.